agent/consul/leader_connect_ca.go

```diff
@@ -417,6 +418,7 @@ func (c *CAManager) initializeRootCA(provider ca.Provider, conf *structs.CAConfi
 	if err != nil {
 		return fmt.Errorf("error generating intermediate cert: %v", err)
 	}
+	interPEM = ca.AddSingleNewline(interPEM)
```

Comment:
same here, this isn't necessary, right?

We might need it later if we store it `interPEM`?